derive/src/lib.rs

```diff
@@ -360,7 +367,19 @@ pub fn compact_as_derive(input: proc_macro::TokenStream) -> proc_macro::TokenStr
 	wrap_with_dummy_const(input, impl_block)
 }
 
-/// Derive `MaxEncodedLen`.
+/// Derive `parity_scale_codec::MaxEncodedLen` for struct and enum.
+///
+/// # Top level attribute
+///
+/// By default the macro will try to bound the types needed to implement `MaxEncodedLen`, but the
+/// bounds can be specified manually with the top level attribute:
+/// ```
+/// # use parity_scale_codec_derive::Encode;
+/// # use parity_scale_codec::MaxEncodedLen;
+/// # #[derive(Encode, MaxEncodedLen)]
+/// #[codec(mel_bound(T: MaxEncodedLen))]
+/// # struct MyWrapper<T>(T);
+/// ```
```

Comment:
yes it may be overkill to set up a compiling example just to check the syntax of the attribute, but it renders fine I think

Reply:
Compiled examples are never an overkill 👍